app/assets/stylesheets/template.css.scss

```diff
@@ -20,3 +20,15 @@ body {
   background-color: #f5f5f5;
   padding: 15px;
 }
```

Comment:
I notice that the green + button doesn't align properly to properties which have a language select included. You could add a rule to override the vertical-align and force the button to align to the top which should look right for fields with and without the additional language select.

.input-group-btn { vertical-align: top; }